cannon/mipsevm/exec/mips_syscalls.go

```diff
@@ -131,8 +132,10 @@ const (
 
 // Other constants
 const (
-	SchedQuantum = 100_000
-	BrkStart     = 0x40000000
+	SchedQuantum              = 100_000
+	BrkStart                  = 0x40000000
+	HZ                        = 10_000_000
```

Comment:
If cannon was optimized, this value would go up?

Reply:
Yes we should increase this in that case. A too low `HZ` will mean the main goroutine is more likely to be eligible to perform a gc assist. The gc assist is still guarded by the heap usage, so it won't happen in most cases. But it would mean we run a couple more instructions to check.

Reply:
It would be super helpful if you put as much context as comments in the code as possible